src/content/en/updates/2020/10/http-cache-partitioning.md

```diff
@@ -0,0 +1,260 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Chrome's http cache partitioning helps with better security and privacy.
+
+{# wf_published_on: 2020-10-01 #}
+{# wf_updated_on: 2020-10-01 #}
+{# wf_featured_image: /web/updates/images/generic/sd-card.png #}
+{# wf_tags: storage-isolation, cache #}
+{# wf_featured_snippet: Chrome's http cache partitioning helps with better security and privacy. #}
+{# wf_blink_components: Blink>Network #}
+
+# Gaining security and privacy by partitioning the cache {: .page-title }
+
+{% include "web/_shared/contributors/agektmr.html" %}
+
+In general, caching can improve performance by storing data so future requests
+for the same data are served faster. For example, a cached resource from the
+network can avoid a round trip to the server. A cached computational result can
+omit the time to do the same calculation.
+
+In Chrome, the cache mechanism is used in various ways and HTTP Cache is one example.
+
+## How Chrome's HTTP Cache currently works
+
+As of version 85, Chrome caches resources fetched from the network, using their
+respective resource URLs as the cache key. (A cache key is used to identify a
+cached resource.)
```

Comment:
```suggestion
cached resource.) A cache key is  a tuple of URLs rather than a single URL.
```

Reply:
This goes with the next request to make it clear what the key is and how it works.

Reply:
OK, I see what you mean. Please have a look at how this will look on the website: https://pr-9038-dot-web-central.appspot.com/web/updates/2020/10/http-cache-partitioning
I'm consistent on the tuples below the image while `+` on the paragraph to the right.

Reply:
At this point we are only talking about status quo which is not using tuples yet. If we were to add this phrase, somewhere after "How will cache partitioning affect Chrome's HTTP Cache?".

Reply:
Yes.

Reply:
Yes about where to put the tuple explanation.